### ISSUE TITLE
in DELETE /v2/entities/<id>/attrs/<attr_name> if try to remove an attr, but with empty name, returns 405 without response payload

### ISSUE THREAD

**Issue by iariasleon** [telefonicaid/fiware-orion#2078]
in `DELETE /v2/entities/<id>/attrs/<attr_name>` if try to remove an attribute, but with empty name, returns 405 without response payload.

#### Remove attribute request
```
DELETE http://localhost:1026/v2/entities/room/attrs/
Content-Type: application/json
Fiware-Service: test_attribute_name_update_error
Fiware-ServicePath: /test
```

#### remove response
```
http code: 405
date: Mon, 25 Apr 2016 08:05:29 GMT
fiware-correlator: 79ca73bc-0abc-11e6-ab91-005056a20feb
connection: Keep-Alive
content-length: 0
allow: GET, PATCH, POST, PUT
```

#### expected response
```
http code: 405
...
{"error": "methodNotAllowed", "description": "not defined yet"}
```

**Comment by kzangeli** [comment]
This is like this (without payload) for 'historical reasons'. The OMA spec defines that responses with **405 Method not allowed** should not have any payload. 

However, for V2 we have no reason to follow that spec and should instead be true to V2 about always returning payload for errors.

**Comment by kzangeli** [comment]
Payload for 405 Method not allowed implemented in #2318
So, this issue is about DELETE /v2/entities/<id>/attrs, and as that request doesn't allow DELETE, a 405 is returned and PR #2318 solves this issue as well (#2318 was implemented to fix #2075)

**Comment by fgalan** [comment]
Fixed by PR #2318. Please @iariasleon have a look and close it if you find it solved.

**Comment by iariasleon** [comment]
LGTM
```
http code: 405 - Method Not Allowed
headers:
   content-length: 63
   fiware-correlator: c0e31f4a-4e6b-11e6-bf27-005056a20feb
   connection: Keep-Alive
   allow: GET, PATCH, POST, PUT
   date: Wed, 20 Jul 2016 11:18:58 GMT
   content-type: application/json
payload: {"error":"MethodNotAllowed","description":"method not allowed"}
```
